docs/guides/multiple-documents.md

```diff
@@ -0,0 +1,107 @@
+# Serving multiple OAS documents
+
+A single Prism instance serves one OAS document.
+In order to serve multiple documents, you can use `docker-compose` utility.
+
+## Preparing configuration files
+
+Depending on your needs, you might have a `docker-compose.yaml` file like this:
+
+```yaml
+version: "3"
+services:
+  proxy:
+    image: nginx:alpine
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - "8080:80"
+    depends_on:
+      - prism_1
+      - prism_2
+  prism_1:
+    image: stoplight/prism:3
+    command: mock -p 4010 --host 0.0.0.0 https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml
+  prism_2:
+    image: stoplight/prism:3
+    command: mock -p 4010 --host 0.0.0.0 https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml
+```
+
+And the corresponding `nginx.conf` file:
+
+```
+worker_processes auto;
+
+events {
+  worker_connections 512;
+}
+
+http {
+  server {
+    listen 80;
+
+    location /app_1 {
+      rewrite ^/app_1/(.*) /$1 break;
+      proxy_pass http://prism_1:4010/;
+    }
+
+    location /app_2 {
+      rewrite ^/app_2/(.*) /$1 break;
+      proxy_pass http://prism_2:4010/;
+    }
+  }
+}
+```
+
+This configuration will allow access to the first Prism instance on `localhost:8080/app_1` and the second instance on `localhost:8080/app_2` endpoint.
+
+## Running the cluster
+
+Let's first place the aforementioned files under the same directory - here `prism`.
+
+```
+prism
+├── docker-compose.yaml
+└── nginx.conf
+```
+
+With the files in place, you can run (from under `prism/`): `docker-compose up`.
+After a second, you should be set up with two Prism instances serving different OAS documents.
+
+## Testing the cluster
+
+If you used `nginx.conf` configuration provided in this document, you should now have two endpoints available:
+
+```
+1. http://localhost:8080/app_1
+2. http://localhost:8080/app_2
+```
+
+You can now try these out by issuing:
+
+```
+1. curl localhost:8080/app_1/pets
+2. curl localhost:8080/app_2/pets
```

Comment:
If I recall correctly the OAS2 document serves `pet/` while the OAS3 serves `pets`

Reply:
don't know, but these 2 will consistently list pets

Reply:
You're right.